concourse/model/traits/image_scan.py

```diff
@@ -143,6 +144,12 @@ class Notify(enum.Enum):
         type=ProtecodeScanCfg,
         doc='if present, perform protecode scanning',
     ),
+    AttributeSpec.optional(
+        name='clam_av',
```

Comment:
I'd prefer for `clam_av` to also feature configuration options (similar to protecode)

Reply:
as discussed (a bit earlier today), use `scanning_mode` or sth. similar, which should be an enum, with one value 'DEFAULT/default', which should be the default (for now)

Reply:
As `clam_av` now has a configuration (albeit one whose values are not used) I've opted not to create a default scanning for now.